src/PeopleSDKAdapter.js

```diff
@@ -54,15 +54,21 @@ export default class PeopleSDKAdapter extends PeopleAdapter {
   }
 
   /**
-   * Returns an observable that emits person data of the access token bearer.
+   * Returns an observable that emits only once of person data of the access token bearer.
    *
    * @public
    * @returns {Observable.<Person>}
    * @memberof PeopleSDKAdapter
    */
   getMe() {
-    // person ID must be retrieved in order to invoke `getPerson` method properly.
-    return from(this.datasource.people.get('me')).pipe(flatMap(({id}) => this.getPerson(id)));
+    return from(this.fetchPerson('me')).pipe(
+      flatMap((person) =>
+        from(this.datasource.internal.presence.get([person.id])).pipe(
+          map((status) => ({...person, status: this.getStatus(status.status)}))
```

Comment:
If you call it `presence` it would read better
```suggestion
          map((presence) => ({...person, status: this.getStatus(presence.status)}))
```